File: data/museum/oseg_generated/java/BuyMuseumTickets_general_entry.java
package org.openapimuseum.client.examples;

import org.openapimuseum.client.ApiException;
import org.openapimuseum.client.Configuration;
import org.openapimuseum.client.api.*;
import org.openapimuseum.client.auth.*;
import org.openapimuseum.client.model.*;

import java.io.File;
import java.util.List;
import java.util.Map;

public class BuyMuseumTickets_general_entry
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        var buyMuseumTickets = new BuyMuseumTickets()
            .ticketType(BuyMuseumTickets.TicketTypeEnum.GENERAL)
            .ticketDate("2023-09-07")
            .email("todd@example.com")
            .ticketId(null)
            .eventId(null);

        try
        {
            var apiCaller = new TicketsApi(config);

            var response = apiCaller.buyMuseumTickets(
                buyMuseumTickets
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling Tickets#buyMuseumTickets");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: data/museum/oseg_generated/python/BuyMuseumTickets_general_entry.py
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    buy_museum_tickets = models.BuyMuseumTickets()
    buy_museum_tickets.ticket_type = "general"
    buy_museum_tickets.ticket_date = "2023-09-07"
    buy_museum_tickets.email = "todd@example.com"
    buy_museum_tickets.ticket_id = None
    buy_museum_tickets.event_id = None

    try:
        api_caller = api.TicketsApi(api_client)

        response = api_caller.buy_museum_tickets(
            buy_museum_tickets=buy_museum_tickets,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Tickets#buy_museum_tickets: %s\n" % e)
